UI/volume-control.cpp

```diff
@@ -594,10 +617,12 @@ VolumeMeter::VolumeMeter(QWidget *parent, obs_volmeter_t *obs_volmeter,
 	magnitudeIntegrationTime = 0.3;                  //  99% in 300 ms
 	peakHoldDuration = 20.0;                         //  20 seconds
 	inputPeakHoldDuration = 1.0;                     //  1 second
+	meterThickness = 3;     // Bar thickness in pixels
+	meterFontScaling = 0.8; // Font size is 80% of Widget font-size
 
 	channels = (int)audio_output_get_channels(obs_get_audio());
 
-	handleChannelCofigurationChange();
+	handleCofigurationChange();
```

Comment:
If you're renaming the method, might as well fix the spelling error.

s/handleCofigurationChange/handleConfigurationChange/g